frontend/src/pages/PipelineDetails.tsx

```diff
@@ -166,23 +177,25 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
             <div className={commonCss.page}>
               {selectedTab === 0 && <div className={commonCss.page}>
                 {this.state.graph && <div className={commonCss.page} style={{ position: 'relative', overflow: 'hidden' }}>
-                  <Paper className={css.summaryCard}>
-                    <div style={{ alignItems: 'baseline', display: 'flex', justifyContent: 'space-between' }}>
-                      <div className={commonCss.header}>
-                        Summary
+                  {summaryShown && (
+                    <Paper className={css.summaryCard}>
+                      <div style={{ alignItems: 'baseline', display: 'flex', justifyContent: 'space-between' }}>
+                        <div className={commonCss.header}>
+                          Summary
+                        </div>
+                        <Button onClick={() => this.setState({ summaryShown: !summaryShown })} color='secondary'>
+                          {summaryShown ? 'Hide' : 'Show'}
```

Comment:
On the line above (186), clicking the button should always set the state to false.